app/controllers/books_controller.rb

```diff
@@ -0,0 +1,67 @@
+class BooksController < ApplicationController
+
+  def findBook
```

Comment:
As this is a helper method rather than an action, we should make it private (by moving it to the bottom of the class definition, under a line that says `private`).